charts/zalenium/templates/_pod-template.yaml

```diff
@@ -158,12 +160,11 @@ spec:
   nodeSelector:
     {{ .Values.nodeSelector.key }}: {{ .Values.nodeSelector.value | quote }}
   {{- end }}
-  {{- if .Values.tolerations }}
+  {{- if (.Values.tolerations) and (.Values.tolerations.enabled) }}
   tolerations:
-{{ toYaml .Values.tolerations | indent 4 }}
+    - key: {{ .Values.tolerations.key }}
```

Comment:
This code brings limitation because you can now only add one toleration

Reply:
I did this change to get the build working, without this change minikube test was failing with this error `303.50s$ ./kubernetes/start-zalenium-in-minikube.sh
namespace "zalenium" created
Error: render error in "zalenium/templates/deployment.yaml": template: zalenium/templates/deployment.yaml:15:8: executing "zalenium/templates/deployment.yaml" at <include "zalenium.po...>: error calling include: template: zalenium/templates/_pod-template.yaml:163:16: executing "zalenium.podTemplate" at <.Values.tolerations....>: can't evaluate field enabled in type interface {}`  
More details here: https://travis-ci.org/zalando/zalenium/jobs/528290644

Reply:
Looks like when I rebased, I got it wrong.
Adding this back
```
  tolerations:
{{ toYaml .Values.tolerations | indent 4 }}
  {{- end }}
  {{- if .Values.affinity }}
  affinity:
{{ toYaml .Values.affinity | indent 4 }}
  {{- end -}}
```